core/src/services/cloudService.ts

```diff
@@ -13,9 +34,7 @@ export interface CloudServiceOptions {
 export interface CloudService {
   /**
    * Invoke a deployed Serverless function
-   * @param name Name of function to forget
-   * @param fireAndForget Don't listen for response if true
-   * @param payload Payload to send in invocation
+   * @param params parameters needed to call the invoke
    */
-  invoke<T>(name: string, fireAndForget: boolean, payload?: any, headers?: StringParams): Promise<T>;
+  invoke<T>(params: InvokeRequest): Promise<T>;
```

Comment:
I would call this `invokeOptions` instead of `params`

Reply:
Hi My! Thanks for this feedback, I will implement it in my next commit.

Reply:
Hi My! Thanks for this feedback, it will be applied in my next commit.